runtime/filesystem-http/src/main/java/io/aklivity/zilla/runtime/filesystem/http/internal/HttpWatchService.java

```diff
@@ -50,19 +50,22 @@ public final class HttpWatchService implements WatchService
     private final WatchKey closeKey = new HttpWatchKey();
 
     private final Duration pollInterval;
+    private final String authorization;
     private final Collection<HttpWatchKey> watchKeys;
     private final BlockingQueue<WatchKey> pendingKeys;
     private final ScheduledExecutorService executor;
 
     private volatile boolean closed;
 
     HttpWatchService(
-        HttpFileSystemConfiguration config)
+        HttpFileSystemConfiguration config,
+        String authorization)
```

Comment:
Can we call `config.authorization()` instead of passing `authorization` separately?